parlai/agents/test_agents/dummy_torch_agent.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Test TorchAgent."""
```

Comment:
Do you mind adding a line explaining that this is a mock agent for unit testing purposes, and it's not meant to be actually invoked?

Reply:
yes, and I will probably rename the agent to reflect this